File: api/models/messages.js
var mongoose = require('mongoose'),
	Schema = mongoose.Schema,
	_ = require('underscore'),
	returnJSON = require('./utils').returnJSON;

var messages_schema = new Schema({
 	text: String,
	url: String,
	id: String,
	account_id: Schema.ObjectId,
	status: {type: String, private_access: true},
	active: String
});

mongoose.model('Messages', messages_schema);
module.exports = Messages = mongoose.model('Messages');

Messages.prototype.returnJSON = returnJSON;

File: dal/payparrot_dal/messages.py
from time import time
import datetime
from random import random
from hashlib import sha512
from bson.objectid import ObjectId

from payparrot_dal.base import BaseModel

class Messages(BaseModel):
    _meta = {
        'collection': 'messages',
        'fields': {
            'account_id': {'required': True, 'type': ObjectId},
			'text': {'required': True},
			'url': {'required': True},
			'status': {'required': True, 'default':True},
			'active': {'default': True}
    	}
    }